index.js

```diff
@@ -73,7 +73,7 @@ module.exports = function (source) {
         hasRun = true;
     }
 
-    var name = slash(path.relative(root || this.options.context, this.resourcePath));
+    var name = slash(path.relative(root || this.rootContext, this.resourcePath));
```

Comment:
Thanks, can keep `this.options.context` for backwards compatability please.

```
slash(path.relative(root || this.rootContext || this.options.context, this.resourcePath));
```

Reply:
+1 smart